libraries/azurerm_ad_user.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require 'azurerm_resource'
+
+class AzurermAdUser < AzurermResource
+  name 'azurerm_ad_user'
+  desc 'Verifies settings for an Azure Active Directory User'
+  example <<-EXAMPLE
+    describe azure_ad_user(azureId: 'azure_id') do
```

Comment:
Resource name is wrong. It should be `azurerm_ad_user`.

Reply:
This example is wrong too. Is this the right way to call this resource? If we use a hash I think the constructor will need to change.